apps/web/app/[locale]/kanban/page.tsx

```diff
@@ -47,6 +50,10 @@ const Kanban = () => {
 		{ name: t('common.YESTERDAY'), value: KanbanTabs.YESTERDAY },
 		{ name: t('common.TOMORROW'), value: KanbanTabs.TOMORROW }
 	];
+	const { user } = useAuthenticateUser();
+	const { openModal, isOpen, closeModal } = useModal();
+	const timezone = userTimezone();
+	console.log('time-zone', timezone);
```

Comment:
@Anishali2 console.log?

Reply:
removed..

Reply:
Sir the is configuration in linting in which user have to remove console every time before pushing code but the reason is if I add this configuration I have to remove every single console in the project in which some consoles are useful

Reply:
Yes @Anishali2 , some are useful 100%, maybe we can add some "hint" to linter above those so it ignores them?